### ISSUE TITLE
Marlin 2.0 MKS Sbase Diamond hotend mixing extruder

### ISSUE THREAD

**Issue by budiman7x** [MarlinFirmware/Marlin#11794]
hi, i have a problem when activated mixing extruder, the all three extruder behave crazy, like a 10 times moving so fast. if mixing deactivated using single extruder, it is perfect so fine

i am using titan extruder clone and drv8825 so my step is 800, just perfect, but if activated mixing extruder, i cant calibrate my step mm extruder, even changing number doesnt affect. first layer can get 2.35 mm thick, OMG

i am attach file my config for mixing extruder

[Configuration diamond.zip](https://github.com/MarlinFirmware/Marlin/files/2367849/Configuration.diamond.zip)

if using single extruder all is fine

[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/2367855/Configuration.zip)

![img_20180910_200618_hht](https://user-images.githubusercontent.com/42495352/45315314-442aa980-b55e-11e8-8b39-1b1621cf6800.jpg)
![img_20180910_200652_hht](https://user-images.githubusercontent.com/42495352/45315315-44c34000-b55e-11e8-87f9-39ee3ba80bd3.jpg)
![Uploading IMG_20180910_200844_HHT.jpg…]()

**Comment by coolhoun** [comment]
I am noticing perhaps the very same issue, though my setup is different.  On 1.1.9 only (not 1.1.8 or any prior built back to 1.1.0), my Diamond hotend (mixing extruder) extrudes, perhaps, three times as much as it should?  I haven't measured, but it is quite a bit more.  I have RAMPS and all A4988 drivers.  

Unlike budiman7x, when using one extruder, it still extrudes much more than supposed to.

 I went through configurations in 1.1.9 really closely, steps per mm are the same as I've always used (97).  I don't need or have enabled distinct E factors.  I tried disabling a couple of the new 1.1.9 features, but I couldn't pin the issue down...

[1.1.9 configs and pins.zip](https://github.com/MarlinFirmware/Marlin/files/2367912/1.1.9.configs.and.pins.zip)
